EIPS/eip-4844.md

```diff
@@ -211,7 +215,6 @@ the `TransactionNetworkPayload` version of the transaction also includes `blobs`
 The execution layer verifies the wrapper validity against the inner `TransactionPayload` after signature verification as:
 
 * All hashes in `blob_versioned_hashes` must start with the byte `BLOB_COMMITMENT_VERSION_KZG`
-* There may be at most `MAX_BLOBS_PER_TX` blob commitments in any single transaction.
```

Comment:
What was the motivation behind removing this limitation? (Would also be helpful understanding why it was introduced originally)

Reply:
it was added to bound the size of blob carrying txns in the mempool

but clients have other ways to bound txn size and the overall number of blobs is already limited in a single block

Reply:
It might be helpful to add a paragraph to the rationale section pointing out that mempools can still come with a max blobs per tx limit, similar to their current max size per tx limit. No need to enforce such a limit at the level of chain validity rules though.